### DISCUSSION TITLE
Move iac modules to their own repo

### DISCUSSION THREAD

**Discussion by ntwkninja** [defenseunicorns/delivery-aws-iac#146] [Announcements]
With the exception of the terraform-state-backend, iac modules are located in this repo. This has allowed us to iterate quickly but does not afford the ability to version modules independently. This means any time a tag is cut for this repo it there could be an impact to one or all modules. 

Proposal: 
1. Move each module to it's own github repo to version each subcomponent independently
2. Create a zarf-package-iac repo (DUBBD & CoCoWoW iac root modules / packages) to build / test components coherently
3. Version the zarf packages as an aggregate of what would be decentralized features

Alternatives considered:
- Leave this repo as is / continue to add modules that may or may not apply to DUBBD and/or CoCoWoW
- Fork this repo & maintain the fork for CoCoWoW
- something else

**Comment by RothAndrew** [comment]
+1 on the idea, though I believe each distinct zarf package should get its own repo. Having multiple packages in one repo will make CI a lot harder to get right.

**Reply by ntwkninja** [reply]
Based on this limited amount of participation / feedback:

Decision:

1. We will move each module in this repo to it's own github repo (enables independent versioning)
2. This repo will be used for iac root modules to build / test components coherently

Things not decided:
How iac zarf packages will be handled
